src/components/modals/Dialog.jsx

```diff
@@ -104,7 +116,9 @@ const Dialog = (props) => {
                     {props.footerContent && (
                       <div className="dialog-footer">
                         <Section
-                          border={isOverflowing ? 'top' : undefined}
+                          border={
+                            props.alwaysShowBorder || isOverflowing ? 'top' : undefined
+                          }
```

Comment:
It "might" be worth adding some conditional checks that are computing `isOverflowing` and not run them if `props.alwaysShowBorder` is set (ideally this is a rare prop, but could be a nice add).

Reply:
would definitely be nice to avoid the overflow checks when possible